tests/integration/models/reddit/test_collections.py

```diff
@@ -138,6 +138,16 @@ def test_update_description(self, _):
             collection.mod.update_description(new_description)
             assert new_description == collection.description
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_update_display_layout(self, _):
+        self.reddit.read_only = False
+        uuid = self.NONEMPTY_REAL_UUID
+        new_display_layout = "GALLERY"
+        with self.use_cassette():
+            collection = self.subreddit.collections(uuid)
+            collection.mod.update_display_layout(new_display_layout)
+            assert new_display_layout == collection.display_layout
```

Comment:
It doesn't appear that the object is updated after the layout is changed. Is there supposed to a new instance initialized?

Reply:
Weird. I'm not sure. This function is similar to update_title and update_description, so I assumed it'd function the same without initializing a new object.

Reply:
Nevermind, the collection isn't fetched until the assertion.